packages/web/src/components/basic/URLParamsProvider.js

```diff
@@ -123,7 +123,16 @@ class URLParamsProvider extends Component {
 				: window.location.search;
 
 			if (searchString !== this.searchString) {
-				window.dispatchEvent(new Event('popstate'));
+				let event;
+				if (typeof (Event) === 'function') {
```

Comment:
We can also move this logic to polyfill file so that we can have all the polyfills at one place. Thoughts @lakhansamani ?

Reply:
@jyash97 true, I was not aware of `src/polyfill.js`.

Let me revert this commit and fix it 👍

Reply:
@jyash97 I agree, I was not aware of `polyfill` in `reactivecore`.

Have added PR for same, https://github.com/appbaseio/reactivecore/pull/35
Please check that once. 
If it works well, I will revert this PR.